src/main/java/com/shaft/dsl/webElements/Element.java

```diff
@@ -0,0 +1,33 @@
+package com.shaft.dsl.webElements;
+
+import com.shaft.gui.element.ElementActions;
+import com.shaft.validation.Validations;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Element {
+    By locator;
+    WebDriver driver;
+    ElementActions elementActions;
+
+    public Element(WebDriver driver, By locator) {
+        this.driver = driver;
+        this.locator = locator;
+        elementActions = new ElementActions(this.driver);
+    }
+
+    public void shouldExist()
+    { Validations.verifyThat().element(driver,locator).exists().perform();
```

Comment:
Can we convert those to assertions? Or at least have two variants one clearly named
assertThatElementExists
And another one
veriftThatElementExists
I think that would be more descriptive and flexible to use...

Reply:
Makes sense